Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -437,6 +437,19 @@
   - type: Storage
     capacity: 20
 
+- type: entity
+  parent: ClothingBeltStorageBase
+  id: ClothingBeltSyndieHolster
+  name: syndicate shoulder holster
+  description: A deep shoulder holster capable of holding many types of ballistics.
+  components:
+  - type: Sprite
+    sprite: Clothing/Belt/syndieholster.rsi
+  - type: Clothing
+    sprite: Clothing/Belt/syndieholster.rsi
+  - type: Storage
```

Comment:
Imo, this should have a whitelist preventing anything other than a gun from being stored here. 
Doesn't make much sense for you to be able to store PDAs or other clothing in a holster.

Reply:
> Imo, this should have a whitelist preventing anything other than a gun from being stored here. Doesn't make much sense for you to be able to store PDAs or other clothing in a holster.

Very good point